lte/gateway/c/core/oai/include/mme_app_ue_context.h

```diff
@@ -525,13 +543,19 @@ ue_mm_context_t* mme_ue_context_exists_mme_ue_s1ap_id(
 ue_mm_context_t* mme_ue_context_exists_enb_ue_s1ap_id(
     mme_ue_context_t* const mme_ue_context_p, const enb_s1ap_id_key_t enb_key);
 
+#ifdef __cplusplus
```

Comment:
Separate PR is fine, but it will good if we could do before porting hash table to protobuf map. Because on inclusion of c++ provided features we get lot of errors as some piece of code is in 'C'